roles/confluent.control_center/templates/control-center.properties.j2

```diff
@@ -70,10 +85,25 @@ confluent.controlcenter.connect.default.cluster={% for host in groups['kafka_con
 {% for ansible_group in ksql_cluster_ansible_group_names %}
 {% for host in groups[ansible_group] %}{% if loop.index == 1%}confluent.controlcenter.ksql.{{ ansible_group }}.url={% endif %}{% if loop.index > 1%},{% endif %}{{ ksql_http_protocol }}://{{ host }}:{{ ksql_listener_port }}{% endfor %}
 
+{% if ksql_ssl_enabled|bool | bool %}
+confluent.controlcenter.ksql.{{ ansible_group }}.ssl.truststore.location={{control_center_truststore_path}}
+confluent.controlcenter.ksql.{{ ansible_group }}.ssl.truststore.password={{control_center_truststore_storepass}}
+confluent.controlcenter.ksql.{{ ansible_group }}.ssl.keystore.location={{control_center_keystore_path}}
+confluent.controlcenter.ksql.{{ ansible_group }}.ssl.keystore.password={{control_center_keystore_storepass}}
+confluent.controlcenter.ksql.{{ ansible_group }}.ssl.key.password={{control_center_keystore_keypass}}
+{% endif %}
+
 {% endfor %}
 {% else %}
 confluent.controlcenter.ksql.default.url={% for host in groups['ksql'] %}{% if loop.index > 1%},{% endif %}{{ ksql_http_protocol }}://{{ host }}:{{ ksql_listener_port }}{% endfor %}
 
+{% if ksql_ssl_enabled|bool %}
+confluent.controlcenter.ksql.default.ssl.truststore.location={{control_center_truststore_path}}
```

Comment:
@norwood Can you take a look at this file? it should be:
`confluent.controlcenter.connect.{connect-cluster-name}.ssl.truststore.location`

and 

`confluent.controlcenter.ksql.{ksql-cluster-name}.ssl.truststore.location`
right?